src/Text/Tabular/Base.agda

```diff
@@ -0,0 +1,166 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Fancy display functions for List-based tables
+--
+-- The functions in this module assume some (unenforced) invariants.
+-- If you cannot guarantee that your data respects these invariants,
+-- you should instead use Text.Tabular.List.
```

Comment:
All of the text in the header gets sucked into `Everything.agda` right? Usually I think we just have a summary line in the header, and paragraphs like this are inserted immediately below the header.

Reply:
I just had a look and you only get the first paragraph. As soon as you
encounter an empty line, the rest is dropped.